src/main/java/gregtech/api/block/VariantBlock.java

```diff
@@ -22,7 +24,7 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class VariantBlock<T extends Enum<T> & IStringSerializable> extends Block {
+public class VariantBlock<T extends Enum<T> & IStringSerializable> extends Block implements IWalkingSpeedBonus {
```

Comment:
Would it be preferred to make a new subclass of VariantBlock, such as `WalkingSpeedVariantBlock`, and extend that for the relevant blocks instead? This would allow the other blocks not to calculate the walking bonus all the time.